src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -13,16 +13,16 @@
             </ResourceDictionary.MergedDictionaries>
         </ResourceDictionary>
     </Window.Resources>
-    
+
     <Grid Background="Black">
 
         <Border Margin="0,0,0,44" BorderThickness="0" Background="Black" >
             <avalonedit:TextEditor Name="editText" FontFamily="Consolas" FontSize="10pt" ShowLineNumbers="True" Background="#272822" Foreground="#F8F8F2" HorizontalScrollBarVisibility="Disabled"/>
         </Border>
-
-        <Button Content="{x:Static p:Resources.PythonScriptEditoCancelButton}" Style="{DynamicResource ResourceKey=STextButton}" HorizontalAlignment="Right" Margin="2" Name="button1" VerticalAlignment="Bottom" Width="75" Click="OnCancelClicked" />
-        <Button Content="{x:Static p:Resources.PythonScriptEditorAcceptChangesButton}" Style="{DynamicResource ResourceKey=STextButton}" HorizontalAlignment="Right" Margin="2 2 80 2" Name="button2" VerticalAlignment="Bottom" Width="130" Click="OnAcceptClicked" />
-
+        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
+            <Button Content="{x:Static p:Resources.PythonScriptEditorAcceptChangesButton}" Style="{DynamicResource ResourceKey=STextButton}" Name="button2" Click="OnAcceptClicked" HorizontalAlignment="Right" VerticalAlignment="Bottom" Margin="2"/>
+            <Button Content="{x:Static p:Resources.PythonScriptEditoCancelButton}" Style="{DynamicResource ResourceKey=STextButton}" Name="button1" Click="OnCancelClicked" VerticalAlignment="Bottom" HorizontalAlignment="Right" Margin="2"/>
+        </StackPanel>
```

Comment:
Hi Ben, I upload an image to show how the button will look like after I modify the string length in the Resources.en-US.resx file. Previously why we set Cancel button as button1 I guess it is because we need to know the width of the right button first, and then we can know the outer margin from the left button to the right border of UI in order to avoid overlap. Now I use <Stackpanel> tag, and give the orientation property "horizontal". The children elements will be displayed from left to right. That's why I swapped two buttons.